File: java/2482.java
class Solution {
    public int[][] onesMinusZeros(int[][] grid) {
        int m = grid.length, n = grid[0].length;
        int rs[] = new int[m], cs[] = new int[n];
        for (int i = 0; i < m; i++) {
            for (int j = 0; j < n; j++) {
                rs[i] += grid[i][j];
                cs[j] += grid[i][j];
            }
        }
        for (int i = 0; i < m; i++)
            for (int j = 0; j < n; j++)
                grid[i][j] = rs[i] * 2 - m + cs[j] * 2 - n;
        return grid;
    }
}

File: python3/2482.py
class Solution:
    def onesMinusZeros(self, grid: List[List[int]]) -> List[List[int]]:
        m = len(grid)
        n = len(grid[0])
        rs = [0] * m
        cs = [0] * n
        for i in range(m):
            for j in range(n):
                rs[i] += grid[i][j]
                cs[j] += grid[i][j]
        for i in range(m):
            for j in range(n):
                grid[i][j] = rs[i] * 2 - m + cs[j] * 2 - n
        return grid
